docs/operate/mobility/move-arm/configure-additional.md

```diff
@@ -58,15 +61,26 @@ Then, configure the gripper's frame to describe its position and orientation rel
    }
    ```
 
+1. Check whether your gripper has a kinematics file.
+
+   Some grippers have [kinematics files](/operate/reference/kinematic-chain-config/) that describe the position and orientation of the jaws of the gripper as they move, relative to the gripper's base.
+   This file typically also contains geometries to represent the gripper's volume and avoid collisions between the gripper and its environment.
```

Comment:
```suggestion
   This file typically also contains geometries that represent the gripper's volume to avoid collisions between the gripper and its environment.
```